### ISSUE TITLE
Slider labels break layout at start/end

### ISSUE THREAD

**Issue by baba43** [mantinedev/mantine#1229]
### What package has an issue

@mantine/core

### Describe the bug

When the beginning or the end of the slider is labeled, the labels break out of the layout. 

This should not be so, or should we solve this problem with our own styles?

![image](https://user-images.githubusercontent.com/1674755/162802557-35293812-f3bd-459b-bd2f-870d11012ba8.png)

### In which browser did the problem occur

All

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by rtivital** [comment]
Try adding `whiteSpace: 'nowrap'` styles to mark labels using Styles API

**Comment by baba43** [comment]
> Try adding `whiteSpace: 'nowrap'` styles to mark labels using Styles API

Thanks, that solves the line break problem, but not the general problem of the caption being positioned outside the surrounding layout.

**Comment by rtivital** [comment]
Labels are positioned at the center of the associated mark, it is designed to be this way – there is not any reliable way to make it other way. I can only suggest to not use such large labels.

**Comment by baba43** [comment]
> Labels are positioned at the center of the associated mark, it is designed to be this way – there is not any reliable way to make it other way. I can only suggest to not use such large labels.

Ok, this is up to you. I solved the issue for my application by overriding the styles like this:

```
const MyRangeSlider = styled(RangeSlider)`
  .mantine-Slider-markWrapper {
    &:first-of-type > .mantine-Slider-markLabel {
      transform: unset;
      position: absolute;
      left: -${16/2}px;
    }
    &:last-of-type > .mantine-Slider-markLabel {
      transform: unset;
      position: absolute;
      right: 0;
    }
  }
`;
```
The only bad part here is the magic value that should be resolved from the theme, I guess. 
(Just in case someone is looking for the same issue in the future)

Thanks anyway!

**Comment by rtivital** [comment]
Fixed in 4.1.4
